### DISCUSSION TITLE
Submission error: Err: tuple index out of range!

### DISCUSSION THREAD

**Discussion by Angelsfs** [PaddlePaddle/PaddleSpatial#158] [BaiduKDDCup2022] [answered]
Hello, 

We're having an error when submitting (Err: tuple index out of range!). In my local environment everything works fine for 2 in data (0001in.zip and 0002in.zip, we have created this last one) and we have modified NUM_MAX_RUNS to 2.

Could you check the backend log for me, please?
Team name: Innova Team
Submission time: 2022-06-08 15:37

Thanks in advance.

**Comment by ramber1836** [comment, marked as answer]
<img width="1223" alt="image" src="https://user-images.githubusercontent.com/19561583/172611108-b0a7071e-7c53-4c5d-9cb1-9947cd784de9.png">

**Reply by Angelsfs** [reply]
Hi again and thanks for the screenshot.
In that line I'm reshaping the prediction to have a numpy array of (134, 288 ,1). Is this the correct shape of the output?
I'm trying locally with the evaluation.py file in path PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/ (updated today), is this correct? Does it have the same structure as the evaluation files in the others environments (specifically base env)?

Thank you in advance.

**Reply by ramber1836** [reply]
> Hi again and thanks for the screenshot. In that line I'm reshaping the prediction to have a numpy array of (134, 288 ,1). Is this the correct shape of the output? 

Yes, correct. 

> I'm trying locally with the evaluation.py file in path PaddleSpatial/apps/wpf_baseline_gru/kddcup22-sdwpf-evaluation/paddlepaddle/ (updated today), is this correct? 

Yes, it's correct. 

> Does it have the same structure as the evaluation files in the others environments (specifically base env)?

Yes.
